### ISSUE TITLE
Use -Duser.home

### ISSUE THREAD

**Issue by JamesOlvertone** [zlatinb/muwire-seedbox-daemon#2]
Hello,

is there a way that seedbox handles the -Duser.home settings?
I execute the jar (yes I know its a selfexcuting jar) with
java -Duser.home=/where_ever -jar muwire-seedbox-daemon.jar 
But it ignores it always.

I tried to add user.home=/where_ever in the application.properties didn't work.

The project is spring boot based and searched around in the spring boot docs and found nothing except the properties launcher maybe this is relevant, I dont know.

I opened the executable jar, there is a section Inline script specified in build properties with some JAVA_OPTS... This section is in the sourcefolder in the file inline.conf. Is this the place to set -Duser.home=.... and rebuild.
Is there a more flexible way to do this at runtime?

**Comment by zlatinb** [comment]
The cleanest way is to change `inlined.conf` to include existing `JAVA_OPTS` instead of overwriting them.  Then you pass any options to the jvm at runtime.

```
JAVA_OPTS="${JAVA_OPTS} -XX:+UseParallelGC"
```

**Comment by JamesOlvertone** [comment]
java -Duser.home=... -jar ....
WORKS but there is never something written to this dir thats why I thought it does not work.
Most javaprogs that I use write something to $home/.java (fonts, userprefs,...) or the jvm writes something to this dir (fonts security deployment,... depends on jre-version and usage of special classes like guistuff....)
I found it out when I set -Djava.io.tmpdir=... too, tomcat is writing stuff to the tmp-dir.

Later I tried the inlined.conf and works too.
